### ISSUE TITLE
PHP Parse error:  syntax error, unexpected '[' in code/VersionedReadingMode.php on line 37

### ISSUE THREAD

**Issue by yoyo23** [heyday/silverstripe-versioneddataobjects#14]
I get a Parse error  unexpected '[' with PHP 5.3.29 in code/VersionedReadingMode.php on line 37

I think
```
if (isset(self::$originalReadingMode) && in_array(self::$originalReadingMode, ['Stage', 'Live'])) {
```
should be
```
if (isset(self::$originalReadingMode) && in_array(self::$originalReadingMode, array('Stage', 'Live'))) {
```

**Comment by camspiers** [comment]
The `[]` syntax is PHP 5.4+ syntax. We can change it so it works in 5.3. but please bear in mind that 5.3 is no longer a supported version of PHP, and so you are encouraged to upgrade.

**Comment by camspiers** [comment]
Fixed and released in 1.3.3
